plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/TestHiveMetastoreMetadataQueriesAccessOperations.java

```diff
@@ -501,4 +509,25 @@ private void assertMetastoreInvocations(@Language("SQL") String query, Multiset<
     {
         assertMetastoreInvocationsForQuery(getDistributedQueryRunner(), getQueryRunner().getDefaultSession(), query, expectedInvocations);
     }
+
+    @Test
+    public void testDescViewWithColumnComment()
+    {
+        String tableName = "default.test_with_column_comment" + randomNameSuffix();
+        String viewName = "default.test_view_with_column_comment" + randomNameSuffix();
+
+        hiveHadoop.runOnHive(format("CREATE TABLE %s(id int, name string) PARTITIONED BY (ds date)", tableName));
+        hiveHadoop.runOnHive(format("CREATE VIEW %s(id, name COMMENT 'test comment', ds COMMENT 'test comment') AS SELECT * FROM %s", viewName, tableName));
+
+        assertTableColumnNames(viewName, "id", "name", "ds");
+        assertQuery(SESSION,
+                "DESC " + viewName,
+                "VALUES " +
+                        "('id',   'integer', '', ''), " +
+                        "('name', 'varchar', '', 'test comment')," +
+                        "('ds',   'date',    '', 'test comment')");
+
+        assertUpdate(SESSION, format("DROP VIEW %s", viewName));
+        assertUpdate(SESSION, format("DROP TABLE %s", tableName));
```

Comment:
SESSION is redundant. We can use + operator in this case. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#prefer-string-formatting